examples/src/main/AndroidManifest.xml

```diff
@@ -137,6 +136,14 @@
                 android:value=".UIActivity"/>
         </activity>
 
+        <activity android:name=".core.GuidanceViewActivity"
+            android:label="@string/title_guidance_view"
+            android:screenOrientation="portrait">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value=".UIActivity"/>
```

Comment:
Shouldn't this go under .CoreActivity? if you look at the android:name value it's .core.GuidanceViewActivity
if you compare with android:name=".ui.ArrivalUiBuildingExtrusionLayerActivityKt